src/assets/scss/components/hero.scss

```
@@ -29,8 +29,12 @@
     }
 
     .content-container {
-        padding: var(--space-xl-2xl) 0;
+        padding: var(--space-xl-4xl) 0 var(--space-2xs);
```

I'm not sure if this is correct (not quite sure how we use all the spacers) but it seems like this gives us the correct hero height to match the current site.

```suggestion
        padding: var(--space-xl-4xl) 0 var(--space-l);
```

---

In fact, I can't figure out the purpose and scope of these spacers from their naming, nor from anywhere else. I just pick the right ones from there.